include/gz/physics/FeatureList.hh

```diff
@@ -22,6 +22,14 @@
 
 #include <gz/physics/Feature.hh>
 
+#if defined(_MSC_VER)
+#pragma warning(push)
+/// Suppress warnings about "base-class is already a base-class"
+/// Typically, this would indicate a diamond pattern in inheritance,
+/// but this is a false positive for the plugin mechanism.
```

Comment:
If I were to be pedantic, the warning is correct that a base class is being repeated so it's not technically a false positive.

However, the repeated base classes in this case have no fields in them, so there's no risk of ambiguity and therefore won't cause any problems. The only reason the base classes get repeated is to coerce some recursive template magic into compiling; they're not doing anything beyond that.

Reply:
Updated the comment to reflect this.